Fleury/Fleury.java

```diff
@@ -1,15 +1,13 @@
 // This code is contributed by Riyazul555
 
-// Time Complexity  =  O(V + E)  where V is the number of vertices and E is the number of edges.
-// Space Complexity =  O(V + E)  where V is the number of vertices and E is the number of edges.
+// Time Complexity  =  O(V + E)  
```

Comment:
No need to remove the legend of the symbols :)

Reply:
You mean you need those comments of what  V and E indicates
Right @iuliagroza ??